plugin.js

```diff
@@ -682,189 +496,423 @@
 		};
 	}
 
-	placeholdersCollection = new PlaceholdersCollection();
+	/**
+	 * @class
+	 * @constructor
+	 * @param {CKEDITOR.editor} editor
+	 */
+	function PlaceholderElementsPlugin(editor)
+	{
+		var _this = this;
```

Comment:
I see. I'm usually only using such techniques when there's actually an "issue", like with the `dispatchEvent` closure.